setup.py

```diff
@@ -1,10 +1,11 @@
+import distutils
 import os
 
 from setuptools import setup
 
 setup(
     name='algorithmia',
-    version='1.4.0',
+    version='1.4.4',
```

Comment:
Since we're adding new functionality, it seems like a minor version bump (`1.4.0` -> `1.5.0`) would be more appropriate given semver - thoughts @ZachEddy ?

Reply:
Yep, I agree. Bumping the minor version makes sense here.

Sidenote: weirdly, this is the only Slack notification that GitHub sent me for this pull request, so I've been missing tons of activity 😕

Reply:
I wanted to wait to do that bump until we are fully tested as we can only use 1.5.0 once in Pypi, but I will add this to the commit here.